node/src/GlideClient.ts

```diff
@@ -358,15 +372,18 @@ export class GlideClient extends BaseClient {
      */
     public async configGet(
         parameters: string[],
-    ): Promise<Record<string, string>> {
-        return this.createWritePromise(createConfigGet(parameters));
+        decoder?: Decoder,
+    ): Promise<Record<string, GlideString>> {
```

Comment:
I assume that config parameter names are defined by the server, so all of them are simple strings.
Can I assume that parameter values are simple strings as well?